wagtailautocomplete/wagtail_hooks.py

```diff
@@ -1,4 +1,8 @@
-from django.contrib.staticfiles.templatetags.staticfiles import static
+try:
+    from django.contrib.staticfiles.templatetags.staticfiles import static
+except ModuleNotFoundError:
+    from django.templatetags.static import static
```

Comment:
> As `static` can be imported from `django.templatetags.static` since the 1.10 Django version - see the [release note](https://docs.djangoproject.com/en/3.0/releases/1.10/#django-contrib-staticfiles) - and wagtail-autocomplete requires at least Django 1.11, the import from `django.contrib.staticfiles.templatetags.staticfiles` can be removed.

Reply:
Ah, that's even better! Done